agate-rest/src/main/java/org/obiba/agate/web/rest/config/ConfigurationResource.java

```diff
@@ -199,4 +199,14 @@ public Response getTranslation(@PathParam("locale") String locale, @QueryParam("
     return Response.ok(
       configurationService.getTranslations(locale, _default).toString(), "application/json").build();
   }
+
+  @GET
+  @Path("/i18n/custom/{locale}.json")
+  @Produces("application/json")
+  public Response getCustomTranslation(@PathParam("locale") String locale) throws IOException {
```

Comment:
It's not the custom translations we are interested in, it's the translations of the user profile fields.